checks.go

```diff
@@ -102,19 +103,23 @@ func runCheck(cond cond) bool {
 	debug("Running condition:\n", cond)
 
 	not := "Not"
+	regex := "Regex"
 	condFunc := ""
 	negation := false
+	cond.Regex = false
 
 	// Ensure that condition type is a valid length
-	if len(cond.Type) <= len(not) {
+	if len(cond.Type) <= len(regex) {
```

Comment:
That's unintuitive but should be valid. I think. Might catch us later if we have a very short check type name